yucca/planning/YuccaPlanner.py

```diff
@@ -105,8 +117,16 @@ def determine_transpose(self):
 
         assert self.transpose_fw is not None, "no transposition, something is wrong."
 
-    def determine_spacing(self):
-        self.target_spacing = np.median(self.dataset_properties["original_spacings"], 0).tolist()
+    def determine_target_spacing_or_size(self):
+        self._determine_target_spacing_or_size()
+        assert self.target_size is None or self.target_spacing is None, (
+            "only one of target size or target spacing should be specified"
+            f" but both are specified here as {self.target_size} and {self.target_spacing} respectively"
+        )
+
+    def _determine_target_spacing_or_size(self):
```

Comment:
Why do we have to have both a private and a public `determine_target_spacing_or_size`? I dont think it would be bad if we just added the logic of `_determine_target_spacing_or_size` into `determine_target_spacing_or_size`.

Reply:
So the reason for this was that you could then overwrite the private version without overwriting the validation check, but Jakob also pointed this out and I ended up making just 1 public version and an additional function (with completely different name) that validates the variables are fine